<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTarea</class>
 <widget class="QDialog" name="QTarea">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tarea</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(162, 10, 10);
border-color: rgb(121, 147, 81);
background-color: rgb(162, 10, 10);
color: rgb(246, 238, 201);</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLimpiar">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editDescripcion">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboEstados">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chqConFchaFin">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Con fecha fin</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Fecha Creacion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="tareaDiaOrigen">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Fecha finalizacion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="tareaDiaFin">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 238, 201);
color: rgb(162, 10, 10);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
